.travis.yml

```diff
@@ -150,16 +149,6 @@ install:
           export QT_API=pyqt;
       fi
 
-    # Reinstalling the required numpy version as it may be downgraded.
-    # TODO It may be removed when/if Sherpa becomes np version agnostic
-    # (maybe as soon as v4.8). See discussion in
-    # https://travis-ci.org/gammapy/gammapy/jobs/99867950#L639
-    - if $SHERPA; then
-          conda install --yes --channel astropy numpy=$NUMPY_VERSION iminuit;
-          conda install --yes --no-pin --channel https://conda.anaconda.org/cxc/channel/dev sherpa;
```

Comment:
Good point about the dev version. Do you think it can be installed painlessly from the github master? As adding back the install from this conda channel doesn't make much sense as it only hosts a 6 month old dev.

I'll try to add another mac build, too.